trulens_eval/DEPRECATION.md

```diff
@@ -1,6 +1,19 @@
 # Deprecation Notes
 
-## Changes in 0.2.0; to be deprecated at 1.0.0
+## Changes in 0.2.2
+
+### Breaking changes to databases
+
+- json serialization of WithClassInfo mixin
+    - key "class_info" renamed to "__tru_class_info"
+
+- json serialization of ObjSerial class
+    - init_kwargs expanded into init_bindings, serialized by a new class
+      Bindings, containing both args and kwargs
+
+## Changes in 0.2.0
+
+### Non-breaking changes to be breaking in 1.0.0
```

Comment:
maybe remove the 1.0.0 version since we don't know the version.

Also; what does non-breaking to be breaking mean?

Reply:
What do you mean we don't know? I'm assuming it is a standard that major version numbers can introduce breaking changes but minor versions cannot hence I'm suggesting that the aliases introduced here (to make the changes non-breaking) will be removed (hence become breaking) at 1.0.0 the earliest.

Reply:
I reworded the points.